libcorn/src/lua.rs

```diff
@@ -0,0 +1,51 @@
+use mlua::prelude::*;
+use crate::Value;
+use mlua::Value as OtherValue;
```

Comment:
Whitespace between use statements and code:

```suggestion
use mlua::Value as OtherValue;

```

There are other fmt issues - please make sure to run `cargo fmt` (and `clippy` if you haven't too)